### DISCUSSION TITLE
chat/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#50] [Comments]
# chat/

点击链接可以和「诗涛」1对话：和诗涛对话
[&hellip;] 目前对基于大模型构建应用还比较模糊，待学习以下内容后再来改进：
[&hellip;] 基于诗涛的博客、笔记构建的问答知识库。&#160;&#x21a9;&#xfe0e;

https://shitao5.org/chat/

**Comment by hongtaoh** [comment]
好厉害！我好奇这个是怎么做到的。

**Reply by Shitao5** [reply]
嘿嘿这个是 [MaxKB](https://github.com/1Panel-dev/MaxKB) ：基于 LLM 大语言模型的知识库问答系统，[项目文档](https://maxkb.cn/docs/)写得挺不错。把自己的博客作为知识库导入，然后接一个大模型的 API 就可以部署了。（用 docker 部署嘎嘎快。）

**Comment by LFG12** [comment]
涛哥太厉害了，在互联网上收拾markdown文件的拆分找到了这里，然后看了涛哥的介绍，如此的丰富且优秀，实在我辈楷模，无比敬佩，
看到退伍后能迅速融入校园，刻苦学习保持到专业第一属实是下了很大的努力
还很有自己的想法，跟随着自己的内心，成为了一名R语言大佬（虽然我没学过R语言）如此多的生活经历的历练，一定会越来越好
向大佬学习，也向您致敬

**Reply by Shitao5** [reply]
嘿嘿希望能帮到您拆分 markdown 文件。属实过奖啦，俺都被吹到天上飞来飞去了哈哈，一起加油，一定会越来越好。
